2020-12-21 18:21:31,173 - celery.utils.dispatch.signal - ERROR - Signal handler <function setup_loggers at 0x0000023BF90A01F0> raised: gaierror(10109, 'getaddrinfo failed')
Traceback (most recent call last):
  File "C:\Users\azzam\miniconda3\envs\GeekerBot2\lib\site-packages\celery\utils\dispatch\signal.py", line 276, in send
    response = receiver(signal=self, sender=sender, **named)
  File "D:\PYTHON KU\GeekerBot2\celerys\worker.py", line 29, in setup_loggers
    slh = logging.handlers.SysLogHandler(address=('logsN.papertrailapp.com', '...'))
  File "C:\Users\azzam\miniconda3\envs\GeekerBot2\lib\logging\handlers.py", line 829, in __init__
    ress = socket.getaddrinfo(host, port, 0, socktype)
  File "C:\Users\azzam\miniconda3\envs\GeekerBot2\lib\socket.py", line 918, in getaddrinfo
    for res in _socket.getaddrinfo(host, port, family, type, proto, flags):
socket.gaierror: [Errno 10109] getaddrinfo failed
2020-12-21 18:21:31,789 - celery.worker.consumer.connection - INFO - Connected to redis://127.0.0.1:6380/0
2020-12-21 18:21:31,803 - celery.worker.consumer.mingle - INFO - mingle: searching for neighbors
2020-12-21 18:21:32,858 - celery.worker.consumer.mingle - INFO - mingle: all alone
2020-12-21 18:21:32,930 - celery.apps.worker - INFO - celery@LAPTOP-47OQ9IM6 ready.
2020-12-21 18:21:35,953 - celery.utils.dispatch.signal - ERROR - Signal handler <function setup_loggers at 0x000001ECC733E430> raised: gaierror(10109, 'getaddrinfo failed')
Traceback (most recent call last):
  File "C:\Users\azzam\miniconda3\envs\GeekerBot2\lib\site-packages\celery\utils\dispatch\signal.py", line 276, in send
    response = receiver(signal=self, sender=sender, **named)
  File "D:\PYTHON KU\GeekerBot2\celerys\worker.py", line 29, in setup_loggers
    slh = logging.handlers.SysLogHandler(address=('logsN.papertrailapp.com', '...'))
  File "C:\Users\azzam\miniconda3\envs\GeekerBot2\lib\logging\handlers.py", line 829, in __init__
    ress = socket.getaddrinfo(host, port, 0, socktype)
  File "C:\Users\azzam\miniconda3\envs\GeekerBot2\lib\socket.py", line 918, in getaddrinfo
    for res in _socket.getaddrinfo(host, port, family, type, proto, flags):
socket.gaierror: [Errno 10109] getaddrinfo failed
2020-12-21 18:21:36,373 - celery.utils.dispatch.signal - ERROR - Signal handler <function setup_loggers at 0x00000277B5E1E430> raised: gaierror(10109, 'getaddrinfo failed')
Traceback (most recent call last):
  File "C:\Users\azzam\miniconda3\envs\GeekerBot2\lib\site-packages\celery\utils\dispatch\signal.py", line 276, in send
    response = receiver(signal=self, sender=sender, **named)
  File "D:\PYTHON KU\GeekerBot2\celerys\worker.py", line 29, in setup_loggers
    slh = logging.handlers.SysLogHandler(address=('logsN.papertrailapp.com', '...'))
  File "C:\Users\azzam\miniconda3\envs\GeekerBot2\lib\logging\handlers.py", line 829, in __init__
    ress = socket.getaddrinfo(host, port, 0, socktype)
  File "C:\Users\azzam\miniconda3\envs\GeekerBot2\lib\socket.py", line 918, in getaddrinfo
    for res in _socket.getaddrinfo(host, port, family, type, proto, flags):
socket.gaierror: [Errno 10109] getaddrinfo failed
2020-12-21 18:22:48,545 - celery.worker.consumer.connection - INFO - Connected to redis://127.0.0.1:6380/0
2020-12-21 18:22:48,577 - celery.worker.consumer.mingle - INFO - mingle: searching for neighbors
2020-12-21 18:22:49,767 - celery.worker.consumer.mingle - INFO - mingle: all alone
2020-12-21 18:22:49,928 - celery.apps.worker - INFO - celery@LAPTOP-47OQ9IM6 ready.
2020-12-21 18:37:23,675 - celery.worker.consumer.connection - INFO - Connected to redis://127.0.0.1:6380/0
2020-12-21 18:37:23,694 - celery.worker.consumer.mingle - INFO - mingle: searching for neighbors
2020-12-21 18:37:24,761 - celery.worker.consumer.mingle - INFO - mingle: all alone
2020-12-21 18:37:24,868 - celery.apps.worker - INFO - celery@LAPTOP-47OQ9IM6 ready.
2020-12-21 18:49:04,234 - celery.worker.consumer.connection - INFO - Connected to redis://127.0.0.1:6380/0
2020-12-21 18:49:04,280 - celery.worker.consumer.mingle - INFO - mingle: searching for neighbors
2020-12-21 18:49:05,333 - celery.worker.consumer.mingle - INFO - mingle: all alone
2020-12-21 18:49:05,372 - celery.apps.worker - INFO - celery@LAPTOP-47OQ9IM6 ready.
2020-12-21 18:58:22,069 - celery.worker.consumer.connection - INFO - Connected to redis://127.0.0.1:6380/0
2020-12-21 18:58:22,083 - celery.worker.consumer.mingle - INFO - mingle: searching for neighbors
2020-12-21 18:58:23,137 - celery.worker.consumer.mingle - INFO - mingle: all alone
2020-12-21 18:58:23,265 - celery.apps.worker - INFO - celery@LAPTOP-47OQ9IM6 ready.
2020-12-21 19:10:34,337 - celery.worker.consumer.connection - INFO - Connected to redis://127.0.0.1:6380/0
2020-12-21 19:10:34,356 - celery.worker.consumer.mingle - INFO - mingle: searching for neighbors
2020-12-21 19:10:35,421 - celery.worker.consumer.mingle - INFO - mingle: all alone
2020-12-21 19:10:35,515 - celery.apps.worker - INFO - celery@LAPTOP-47OQ9IM6 ready.
2020-12-21 19:16:25,985 - celery.worker.consumer.connection - INFO - Connected to redis://127.0.0.1:6380/0
2020-12-21 19:16:26,013 - celery.worker.consumer.mingle - INFO - mingle: searching for neighbors
2020-12-21 19:16:27,064 - celery.worker.consumer.mingle - INFO - mingle: all alone
2020-12-21 19:16:27,237 - celery.apps.worker - INFO - celery@LAPTOP-47OQ9IM6 ready.
2020-12-21 19:19:30,865 - celery.worker.consumer.connection - INFO - Connected to redis://127.0.0.1:6380/0
2020-12-21 19:19:31,139 - celery.worker.consumer.mingle - INFO - mingle: searching for neighbors
2020-12-21 19:19:32,200 - celery.worker.consumer.mingle - INFO - mingle: all alone
2020-12-21 19:19:32,235 - celery.apps.worker - INFO - celery@LAPTOP-47OQ9IM6 ready.
2020-12-21 19:24:17,898 - celery.worker.consumer.connection - INFO - Connected to redis://127.0.0.1:6380/0
2020-12-21 19:24:17,915 - celery.worker.consumer.mingle - INFO - mingle: searching for neighbors
2020-12-21 19:24:18,962 - celery.worker.consumer.mingle - INFO - mingle: all alone
2020-12-21 19:24:19,049 - celery.apps.worker - INFO - celery@LAPTOP-47OQ9IM6 ready.
2020-12-21 19:32:44,555 - celery.worker.consumer.connection - INFO - Connected to redis://127.0.0.1:6380/0
2020-12-21 19:32:44,611 - celery.worker.consumer.mingle - INFO - mingle: searching for neighbors
2020-12-21 19:32:45,724 - celery.worker.consumer.mingle - INFO - mingle: all alone
2020-12-21 19:32:45,760 - celery.apps.worker - INFO - celery@LAPTOP-47OQ9IM6 ready.
